governance/third-generation/gcp/restrict-gce-machine-type.sentinel

```diff
@@ -0,0 +1,32 @@
+# This policy uses the Sentinel tfplan/v2 import to require that
```

Comment:
Beautiful.  without comments it will fit on a single slide now

Reply:
Thanks @scarolan